### ISSUE TITLE
Can't install Directus

### ISSUE THREAD

**Issue by martinfojtik** [directus/directus#1650]
## Version Info
  - Directus version and branch: version/6.4
  - PHP version: 7.0.10
  - MySQL version: 5.7.18
  - Web server: WAMP Apache
  - OS name and version: Win 10

## Expected Behavior
Show first installation step.

## Actual Behavior
I have fresh Directus in subfolder on my server, if I open `http://myweb.com/admin/` I am automatically redirected to `http://admin/installation/index.php?step=0` (URL is missing domain name)

## Steps to Reproduce
 1. 
 2. 
 3.

**Comment by WellingGuzman** [comment]
Thanks @martinfojtik, it looks like bug (Directus) while redirecting on windows machine, we would take a look when we have the change to use a WAMP server, in the moment try `http://myweb.com/admin/installation/index.php?step=0`

**Comment by martinfojtik** [comment]
Thanks @WellingGuzman for quick response. Can you try this fix on Linux machine? 

In `api/core/functions.php` change:

```
$path = trim($path, '/');
$subPath = ltrim($subPath, '/');
```

to:

```
$path = trim($path, DIRECTORY_SEPARATOR);
$subPath = ltrim($subPath, DIRECTORY_SEPARATOR);
```
I am not sure if this will work on Linux.

**Comment by WellingGuzman** [comment]
@martinfojtik does that change fixed it for you? if it does, please send us a PR with those changes, in my end it didn't make any different and it shouldn't (hopefully).

**Comment by WellingGuzman** [comment]
Closed by #1662
